VARIABLES.md

```diff
@@ -278,15 +278,15 @@ Default:  false
 
 ### zookeeper_user
 
-Only use to customize Linux User Zookeeper Service runs with. User must exist on host.
+Only use to customize Linux User Zookeeper Service runs with. Default user is cp-kafka
```

Comment:
Only use to customize Linux User that the Zookeeper Service runs with. Default user is cp-kafka.